deebot_client/map.py

```diff
@@ -234,8 +229,13 @@ def _draw_map_pieces(self, draw: ImageDraw.ImageDraw) -> None:
                                 point_y,
                             )
                             raise MapError("Map Limit reached!")
-                        if pixel_type in [0x01, 0x02, 0x03]:
+                        if pixel_type in _COLOR_LIST_INCLUDE:
                             draw.point((point_x, point_y), fill=_COLORS[pixel_type])
+                        elif (
+                            pixel_type not in _COLOR_LIST_INCLUDE
+                            and pixel_type not in _COLOR_LIST_EXCLUDE
+                        ):
```

Comment:
```suggestion
                        elif pixel_type not in _COLOR_LIST_EXCLUDE:
```

The first statement can be omitted as it is always True.

Why do we need to draw all remaining values as floor?

Reply:
removed in #373